Fix urban command producing errors

Gonna see if there are any errors later.

Reason for failure:
`[ERROR] Could not execute command: CommandError {
    cmd: Command { key = "urban", desc = "Searches a term on UrbanDictionary", help = "<term>", cat = "Fun" },
    err: "could not get urban mug: expected value at line 1 column 1",
}
`

--- a/src/commands/fun/urban.rs
+++ b/src/commands/fun/urban.rs
@@ -50,7 +50,8 @@ fn urban_command(args: CommandArguments) -> CommandResult {
     uo.example = uo.example.replace("]", "");
 
 
-    let mug = unwrap_cmd_err!(&URBAN_COMMAND, super::urbanmug::get_mug(&term), "could not get urban mug");
+    let mug = super::urbanmug::get_mug(&term).unwrap_or_default();
+
     let mock_mug_filename = format!("{}.jpg", uo.defid);
 
     let result = args.m.channel_id.send_message(args.ctx, |mb| {
@@ -68,7 +69,9 @@ fn urban_command(args: CommandArguments) -> CommandResult {
 
             eb
         });
+
         mb.add_file(AttachmentType::Bytes((&mug, &mock_mug_filename)));
+
         mb
     });
 

--- a/src/commands/fun/urbanmug.rs
+++ b/src/commands/fun/urbanmug.rs
@@ -22,7 +22,13 @@ fn mug_command(args: CommandArguments) -> CommandResult {
 
     let s = split[1..split.len()].join(" ");
 
-    let mug = unwrap_cmd_err!(&URBANMUG_COMMAND, get_mug(&s), "could not get mug from urban dictionary");
+    let mug = match get_mug(&s) {
+        Ok(s) => s,
+        Err(_) => {
+            let _ = args.m.reply(args.ctx, "Sorry, it looks like UrbanDictionary has some problems... Please try again later!");
+            return Ok(MarkAsFailed);
+        }
+    };
 
     let _ = args.m.channel_id.send_message(args.ctx, |mb| {
         mb.embed(|mut eb| {

